### PR TITLE
Added new version of linear search algorithm 

### PR DESCRIPTION
**Pull request by PANDATD**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by PANDATD — COMMENTED** [review]
removed unwanted variables fixed bugs added wikipedia link

File: searches/linear_search_v2.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""
+Linear search is sequential search algorithm 
+which helps to find the element linearly.
+More info:
+https://en.wikipedia.org/wiki/Linear_search
+"""
```
Comment (PANDATD) [inline searches/linear_search_v2.py:6]: - [x] Discription of Program

File: searches/linear_search_v2.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    size =len(array)
+    i= 0
+    for i in range(size):
+        if array[i] == key:
+            print(i)
+        else :
+            if i == size-1:
+                print("Not found")
+        i = i+1
```
Comment (PANDATD) [inline searches/linear_search_v2.py:21]: - Main search function

File: searches/linear_search_v2.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    for i in range(size):
+        if array[i] == key:
+            print(i)
+        else :
+            if i == size-1:
+                print("Not found")
+        i = i+1        
+if __name__ == "__main__":
+    import doctest 
+    doctest.testmod(name="search", verbose=True)
```
Comment (PANDATD) [inline searches/linear_search_v2.py:24]: - Test Seacation

Comment (PANDATD) [inline searches/linear_search_v2.py:24]: - Test Section

**Comment by cclauss** [comment]
.github/workflows/codeql-analysis.yml should be removed.

**Comment by cclauss** [comment]
Not better than / different from existing algorithms.  Not black formatted.  Only one happy path test.

**Comment by PANDATD** [comment]
> 
> 
> Not better than / different from existing algorithms. Not black formatted. Only one happy path test.

**Hey i agreeed with your statement but guide me how i progress in open-source controbution **

**Pull request closed without merging**